src/python/aps2.py

```diff
@@ -256,11 +272,9 @@ class APS2(metaclass=Parser):
 	set_channel_enabled  = APS2_Chan_Setter(c_int)
 	get_channel_enabled  = APS2_Chan_Getter(c_int, return_type=bool)
 
-
-	def __init__(self, ip_address):
+	def __init__(self):
```

Comment:
Fair enough, I'll leave the address-specific instantiation to the pycontrol wrapper.